vendor-extra/Views/src/ViewConverterVisitor.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
there is a need for documenting what problem this `Visitor` implementation is solving, to make it scalable for other various patterns and content fragments that will come along